substrate-airgapped/examples/signed_tx_from_pair.rs

```diff
@@ -1,49 +1,46 @@
 use codec::Encode;
-use sp_core::H256;
+use primitive_types::H256;
 use sp_runtime::{generic::Header, traits::BlakeTwo256, DeserializeOwned};
 use substrate_airgapped::{
 	balances::Transfer, CallIndex, GenericCall, KusamaRuntime, MortalConfig, Mortality, Tx,
 	TxConfig,
 };
 
 // Example deps
+use anyhow::anyhow;
 use hex;
 use reqwest;
 use serde::{Deserialize, Serialize};
 use sp_keyring::AccountKeyring;
 use sp_version::RuntimeVersion;
+use std::str::FromStr;
```

Comment:
dq: Where does this trait get used?

Reply:
You need the trait in scope to make `H256`'s implementation(s) of `FromStr` to activate. (The compiler will tell you this, try removing this line and you'll see it).